src/Finder/LscpuFinder.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Fidry CPUCounter Config package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\CpuCoreCounter\Finder;
+
+use function sprintf;
+
+/**
+ * The number of cores.
```

Comment:
```suggestion
 * The number of logical cores or processors (depending of the configuration).
```

Reply:
Comment above has been split into the two finders. Need any change here or we can resolve?